<?xml version="1.0" encoding="UTF-8"?>
<testng-results ignored="0" total="1" passed="0" failed="1" skipped="0">
  <reporter-output>
  </reporter-output>
  <suite started-at="2021-11-06T14:59:42 EET" name="Default suite" finished-at="2021-11-06T14:59:57 EET" duration-ms="15726">
    <groups>
    </groups>
    <test started-at="2021-11-06T14:59:42 EET" name="Default test" finished-at="2021-11-06T14:59:57 EET" duration-ms="15726">
      <class name="Tests.Task1Test">
        <test-method is-config="true" signature="start()[pri:0, instance:Tests.Task1Test@6e20b53a]" started-at="2021-11-06T14:59:33 EET" name="start" finished-at="2021-11-06T14:59:42 EET" duration-ms="8446" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- start -->
        <test-method is-config="true" signature="setUp(java.lang.reflect.Method)[pri:0, instance:Tests.Task1Test@6e20b53a]" started-at="2021-11-06T14:59:42 EET" name="setUp" finished-at="2021-11-06T14:59:42 EET" duration-ms="9" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[public void Tests.Task1Test.test_1()]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setUp -->
        <test-method signature="test_1()[pri:0, instance:Tests.Task1Test@6e20b53a]" started-at="2021-11-06T14:59:42 EET" name="test_1" finished-at="2021-11-06T14:59:57 EET" duration-ms="15385" status="FAIL">
          <exception class="java.lang.IndexOutOfBoundsException">
            <message>
              <![CDATA[Index 2 out of bounds for length 0]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.IndexOutOfBoundsException: Index 2 out of bounds for length 0
at java.base/jdk.internal.util.Preconditions.outOfBounds(Preconditions.java:64)
at java.base/jdk.internal.util.Preconditions.outOfBoundsCheckIndex(Preconditions.java:70)
at java.base/jdk.internal.util.Preconditions.checkIndex(Preconditions.java:248)
at java.base/java.util.Objects.checkIndex(Objects.java:373)
at java.base/java.util.ArrayList.get(ArrayList.java:425)
at Pages.Task1Page.Search(Task1Page.java:41)
at Tests.Task1Test.test_1(Task1Test.java:49)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:567)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:132)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:599)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:174)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:822)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:147)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1507)
at org.testng.TestRunner.privateRun(TestRunner.java:764)
at org.testng.TestRunner.run(TestRunner.java:585)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:384)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:378)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:337)
at org.testng.SuiteRunner.run(SuiteRunner.java:286)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1218)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1140)
at org.testng.TestNG.runSuites(TestNG.java:1069)
at org.testng.TestNG.run(TestNG.java:1037)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.IndexOutOfBoundsException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- test_1 -->
        <test-method is-config="true" signature="results(java.lang.reflect.Method, org.testng.ITestResult)[pri:0, instance:Tests.Task1Test@6e20b53a]" started-at="2021-11-06T14:59:57 EET" name="results" finished-at="2021-11-06T14:59:57 EET" duration-ms="256" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[public void Tests.Task1Test.test_1()]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[[TestResult name=test_1 status=FAILURE method=Task1Test.test_1()[pri:0, instance:Tests.Task1Test@6e20b53a] output={null}]]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- results -->
        <test-method is-config="true" signature="tearDown()[pri:0, instance:Tests.Task1Test@6e20b53a]" started-at="2021-11-06T14:59:58 EET" name="tearDown" finished-at="2021-11-06T15:00:00 EET" duration-ms="2043" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
      </class> <!-- Tests.Task1Test -->
    </test> <!-- Default test -->
  </suite> <!-- Default suite -->
</testng-results>
